Sources/NIOTransportServices/NIOTSListenerChannel.swift

```diff
@@ -339,11 +339,15 @@ extension NIOTSListenerChannel: StateManagedChannel {
     }
 
     public func channelRead0(_ data: NIOAny) {
+        self.eventLoop.assertInEventLoop()
+
         let channel = self.unwrapData(data, as: NIOTSConnectionChannel.self)
         let p: EventLoopPromise<Void> = self.eventLoop.newPromise()
```

Comment:
shouldn't this be for `channel.eventLoop`?

Reply:
As we use `channel.close` in the callback it doesn't *really* matter, but it's definitely faster, so let's do that.

Reply:
Resolved.